Extension/src/LanguageServer/configurations.ts

```diff
@@ -2132,6 +2139,9 @@ export class CppProperties {
                             badPath = `"${expandedPaths[0]}"`;
                         }
                         message = localize('cannot.find', "Cannot find: {0}", badPath);
+                        if (incorrectExpandedPaths.length === 1 && incorrectExpandedPaths[0].match(/".*"/) !== null) {
```

Comment:
Should this be >= 1?

Reply:
I wasn't planning on supporting that case (multiple paths in a single string), but it might be easy enough to do so.

Reply:
Sure, up to you.

Reply:
It turned out to be trivial to support this.